packages/remix-server-runtime/router/router.ts

```diff
@@ -1952,7 +1978,7 @@ export function unstable_createStaticHandler(
     if (!actionMatch.route.action) {
       let href = createHref(new URL(request.url));
       if (isRouteRequest) {
-        throw createRouterErrorResponse(`No action found for [${href}]`, {
+        throw createRouterErrorResponse(null, {
```

Comment:
Align with remix response structure - we have the URL in the request so this isn't actually providing much extra info